src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --active: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  margin: auto;
+
+  @media screen and (min-width: 1024px) {
+    max-width: 1024px;
+  }
+
+  @media screen and (min-width: 1200px) {
+    max-width: 1200px;
+  }
+}
+
+.logo {
+  margin: 0;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0 -20px 0 0;
```

Comment:
still looks redundant

Reply:
if I remove styles from .nav__list, then:
1. Indents appear
2. the menu appears in terms